docs/howto/libdefs.md

```diff
@@ -17,6 +17,39 @@ dependencies well-typed.
 
 -----
 
+## Normal type definitions (`.js.flow`): `sqlite3`
```

Comment:
Yeah! I'm pretty happy with how this turned out.

I think for any of our existing libdefs, too, if we find ourselves spending any time writing changes to it then it'll probably make sense to first move it into this format (because it makes iteration so much more pleasant.) Should be a matter of (a) moving the file, (b) dropping the `declare module 'foo' {` and its `}`, and (c) adding to the flowconfig line. And (a') splitting the file up into one per module, if there are several modules in the libdef file.

That might even go for the libdef files we generate with flowgen. I think even some of the quirks of its output that we've had to fix up manually are because it's intended to be used to generate `.js.flow` files, not libdef files, so that could simplify our use of flowgen.